src/main/java/de/tum/in/www1/artemis/service/connectors/BambooService.java

```diff
@@ -371,8 +369,8 @@ public Result onBuildCompletedOld(ProgrammingExerciseParticipation participation
         result.setParticipation((Participation) participation);
 
         addFeedbackToResult(result, buildResults);
-        // save result, otherwise the next database access programmingSubmissionRepository.findByCommitHash will throw an exception
-        resultRepository.save(result);
+/*        // save result, otherwise the next database access programmingSubmissionRepository.findByCommitHash will throw an exception
+        resultRepository.save(result);*/
```

Comment:
Could you maybe add a comment here on why you have commented out this part?

Reply:
The result is saved manually in the only calling method:
https://github.com/ls1intum/Artemis/pull/695/commits/c32d06eb04d50a8dfcd9f5f3bd097874fd915222#diff-5ffcfdcc92994d424ce48cfa2becd4f4R293

Reply:
This was unintentional, I changed it back.